USca/USca_RTU/Processor/SimulatorNew.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using USca_RTU.Processor.Simulator;
+using USca_Server.Tags;
+
+namespace USca_RTU.Processor
+{
+    public partial class SimulatorNew : INotifyPropertyChanged
+    {
+        private int AddressCnt = 0;
+        private Random r = new();
```

Comment:
Should provide a seed so we have repeatable simulations

Reply:
[abf71c9](https://github.com/magley/untitled-scada-project/pull/25/commits/abf71c91abba5eb5614b158977afb88948a1e6c3)

Reply:
This doesn't compile, make `Seed` const

Reply:
I've made it `static` in case we ever want to (de)serialize the seed from a config.

[9baad77](https://github.com/magley/untitled-scada-project/pull/25/commits/9baad77e0df80c3f0efdca99a6c19f9d6c43c86f)